### DISCUSSION TITLE
Löschen historischer Daten

### DISCUSSION THREAD

**Discussion by derheinrich2019** [solectrus/solectrus#2082] [Q&A] [answered]
Wie kann ich die bisher gesammelten PV-Daten und somit auch Erträge etc. löschen?

**Comment by ledermann** [comment, marked as answer]
SOLECTRUS ist eigentlich dafür ausgelegt, Messwerte dauerhaft zu speichern. Nur dadurch sind Auswertungen möglich, die die Vergangenheit betreffen :-) SOLECTRUS selbst löscht daher an keiner Stelle irgendwelche Messwerte.

Wenn Du aber wirklich sämtliche Messwerte löschen möchtest, dann ist es das einfachste, die Datenbank der InfluxDB manuell zu löschen. Vorher müssen die Container herunter gefahren und danach wieder neu gestartet werden, wodurch die Datenbank neu angelegt wird. Das geht wie folgt, hier am Beispiel einer Raspberry-Installation:

```shell
cd /home/pi/solectrus
docker compose down
rm -rf influxdb
docker compose up
```

**Aber Vorsicht, dann sind wirklich alle Messwerte weg und auch etwaige andere Dinge, die Du möglicherweise in InfluxDB gespeichert hast!** 

Es bleiben nur die Strompreise übrig, da diese nicht in InfluxDB, sondern in der PostgreSQL-Datenbank gespeichert werden.
